packages/maskbook/src/web3/UI/EthereumWalletConnectedBoundary.tsx

```diff
@@ -85,7 +86,12 @@ export function EthereumWalletConnectedBoundary(props: EthereumWalletConnectedBo
     if (!chainIdValid && !offChain)
         return (
             <Grid container>
-                <ActionButton className={classes.button} disabled fullWidth variant="contained" size="large">
+                <ActionButton
```

Comment:
Please avoid using ActionButton. Use the standard Button instead.

Reply:
OK, we will replace `ActionButton` in the future, but not in this PR.